.env

```diff
@@ -53,6 +53,9 @@ LOCAL_DB_TYPE=mysql
 ##
 LOCAL_DB_VERSION=5.7
 
+# Whether or not to enable multisite.
+LOCAL_MULTISITE=false
```

Comment:
With this now available, would it make sense to turn this to `true` for the multisite test jobs for PHPUnit?

If not, then we should at least have some type of test that ensures the environment starts and works when multisite is enabled. That could potentially be a separate workflow that tests all of the related local environment commands, and perhaps better off as a separate ticket.